tests/chart_tests/test_vector.py

```diff
@@ -23,8 +23,10 @@ def vector_common_tests(doc):
 
     def test_vector_defaults(self, kube_version):
         """Test that vector behaves as expected with default values."""
+        values = {"global": {"logging": {"collector": "fluentd"}}}
```

Comment:
If we're giving values in this test then we need to update the test name and docstring